packages/extension/src/companion/CompanionContent.tsx

```diff
@@ -27,14 +27,18 @@ type CompanionContentProps = {
   onViewComments?: (state: boolean) => void;
 };
 
+const PX_PER_REM = 16;
+const COMPANION_TOP_OFFSET_REM = 7.5;
+const COMPANION_TOP_OFFSET_PX = PX_PER_REM * COMPANION_TOP_OFFSET_REM;
```

Comment:
We already know the value so let's do it this way. It's simpler.

```suggestion
const COMPANION_TOP_OFFSET_PX = 120;
```